# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/xmldecl/README.md
# Commit: 3dfd3c94a105
# Full Hash: 3dfd3c94a105e095aada0b356f1106370de222d3
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-07 15:51:17
# Regressor Bug: 1701828
# File Overlap Count: 1
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/third_party/rust/xmldecl/README.md b/third_party/rust/xmldecl/README.md
--- a/third_party/rust/xmldecl/README.md
+++ b/third_party/rust/xmldecl/README.md
@@ -1,35 +1,38 @@
 # xmldecl
 
-[![Build Status](https://travis-ci.org/hsivonen/xmldecl.svg?branch=master)](https://travis-ci.org/hsivonen/xmldecl)
-[![crates.io](https://meritbadge.herokuapp.com/xmldecl)](https://crates.io/crates/xmldecl)
+[![crates.io](https://img.shields.io/crates/v/xmldecl.svg)](https://crates.io/crates/xmldecl)
 [![docs.rs](https://docs.rs/xmldecl/badge.svg)](https://docs.rs/xmldecl/)
-[![Apache 2 OR MIT dual-licensed](https://img.shields.io/badge/license-Apache%202%20%2F%20MIT-blue.svg)](https://github.com/hsivonen/xmldecl/blob/master/COPYRIGHT)
+[![Apache 2 OR MIT dual-licensed](https://img.shields.io/badge/license-Apache%202%20%2F%20MIT-blue.svg)](https://github.com/hsivonen/xmldecl/blob/main/COPYRIGHT)
 
 `xmldecl::parse()` extracts an encoding from an ASCII-based bogo-XML
 declaration in `text/html` in a Web-compatible way.
 
 See https://github.com/whatwg/html/pull/1752
 
 ## Licensing
 
 Please see the file named
-[COPYRIGHT](https://github.com/hsivonen/xmldecl/blob/master/COPYRIGHT).
+[COPYRIGHT](https://github.com/hsivonen/xmldecl/blob/main/COPYRIGHT).
 
 ## Documentation
 
 Generated [API documentation](https://docs.rs/xmldecl/) is available
 online.
 
 ## Minimum Supported Rust Version
 
 MSRV: 1.51.0
 
 ## Release Notes
 
+### 0.2.0
+
+* Remove the 1024-byte limit.
+
 ### 0.1.1
 
 * Map UTF-16LE and UTF-16BE to UTF-8 as in `meta`.
 
 ### 0.1.0
 
 * The initial release.